front/SettingsForm.tsx

```diff
@@ -79,6 +80,7 @@ const SettingsForm = () => {
           <fieldset className="form-group">
             <textarea
               className="form-control form-control-lg"
+              data-cy="updateBio"
```

Comment:
The `data-cy` attribute for the `updateBio` textarea is correctly added, which is important for Cypress testing.